### DISCUSSION TITLE
Materiais: recursos e ferramentas

### DISCUSSION THREAD

**Discussion by BeyondMagic** [cerejeiros/mobile#6] [General]
@cerejeiros/front-end postem aqui quais materiais vocês usam, ou acham que outros membros possam aproveitar, mesmo que não agora, mas no futuro para outros projetos ou ideias que vocês possam vir ter.

**Ferramentas:**

+ _Material3 Theme Builder_: Teste de paleta de cores (criação de temas): https://m3.material.io/theme-builder#/custom
+ _Leonardo Color Tool_: . https://leonardocolor.io/
+ _Figma_: Para visualização de design de forma mais simplificada: https://www.figma.com/
+ _Inkscape_: Para criação de gráficos em vetores (logos e designs em geral): https://inkscape.org/

**Recursos:**
+ _Material3 Color Theme Creation_: Criação de paleta baseado em conforto: https://m3.material.io/design/color/the-color-system.html#color-theme-creation
+ _Alura_: Para aprendizado das linguagens de programação do projeto.
+ - _Formação Javascript (front-end)_: https://cursos.alura.com.br/formacao-javascript-front-end?preRequirementFrom=typescript
+ - _Formação Typescript_: https://cursos.alura.com.br/formacao-typescript
+ _Dribble Logos With Golden Ratio_: Biblioteca de logos feitas baseadas na proporção áurea: https://dribbble.com/tags/logo_with_golden_ratio
+ _Vimeo 2D Animations_: Coleção de vídeos animados em 2D: 
+ _Logos by Nick (Inkscape)_: Designer gráfico que faz vídeos demonstrando criação de logos: https://www.youtube.com/@LogosByNick
+ _SASS (documentation)_: Documentação do pré-processador de CSS: https://sass-lang.com/documentation/

**Comment by kaioenzo** [comment]
@cerejeiros/front-end 
*Conteúdo para o aprendizado de react*
 - React native express: Site criado para o ensino básico para o desenvolvimento em react native [https://www.reactnative.express](https://www.reactnative.express)
 - Documentação oficial do react: [React dev](https://react.dev)

**Comment by kaioenzo** [comment]
@cerejeiros/front-end Recomendo a leitura da biblioteca de navegação [React Navigation](https://reactnavigation.org/) e também desse tutorial que mostra como configuras as rotas do app [Template de rotas React Native com native stack, bottom-tabs e drawer navigator](https://www.tabnews.com.br/marcosveloso/template-de-rotas-react-native-com-native-stack-bottom-tabs-e-drawer-navigator )

**Reply by kaioenzo** [reply]
@limonadaquente

**Reply by limonadaquente** [reply]
okay!!! obrigada
